src/app/qgsoptions.h

```diff
@@ -228,6 +228,8 @@ class APP_EXPORT QgsOptions : public QgsOptionsDialogBase, private Ui::QgsOption
     void saveContrastEnhancement( QComboBox *cbox, const QString &name );
     void initMinMaxLimits( QComboBox *cbox, const QString &name, const QString &defaultVal );
     void saveMinMaxLimits( QComboBox *cbox, const QString &name );
+    void setZoomFactorValue( QSpinBox *zoomFactorSpinBox );
+    double getZoomFactorValue( QSpinBox *zoomFactorSpinBox);
```

Comment:
We normally don't prefix getters with `get`.

Just call this `zoomFactorValue`